ted2zim/scraper.py

```diff
@@ -140,6 +134,27 @@ def extract_all_video_links(self):
                 tot_videos_scraped += num_videos_extracted
                 page += 1
             logger.info(f"Total video links found in {topic}: {tot_videos_scraped}")
+            if tot_videos_scraped == 0:
+                self.topics.remove(topic)
+                logger.debug(
+                    f"Removed topic {topic} from topic list as it had no videos"
+                )
+        if not self.topics:
+            raise ValueError("Wrong topic(s) were supplied. No videos found")
+        self.update_title_and_description()
+
+    def update_title_and_description(self):
+        if len(self.topics) > 1:
+            if self.title is None:
+                self.title = "TED Collection"
+            if self.description is None:
```

Comment:
`zimwriterfs` doesn't accept empty description so you should test `if not self.description:` instead.
While it accepts empty title, we could harmonize to require one and thus consider empty string on title as missing as well.